conjur/util/init_utils.py

```diff
@@ -21,9 +24,9 @@ def validate_init_action_ssl_verification_input(ca_path, is_self_signed, ssl_ver
         use_ca_bundle = True
     options = [use_ca_bundle, is_self_signed, not ssl_verify]
     if sum(options) > 1:
-        raise MissingRequiredParameterException("SSL verification method can be one of three:"
-                                                "\n1. --ca-cert with '< Full path to RootCA PEM"
-                                                " file >' (recommended)"
+        raise MissingRequiredParameterException("Can't accept more than one of the following "
+                                                "arguments:"
+                                                "\n1. --ca-cert"
                                                 "\n2. --self-signed"
                                                 "\n3. --insecure (skip certificate validation)")
```

Comment:
2 things:
1. The exception type is not correct. there's no required parameter missing, it's the opposite. there are too many parameters.
2. The exception should not contain instructions to the user, especially not in `util` functions, and more especially not instructions for cli user.
To give instructions to the user, catch this exception type in `InitController`, and write it to stdout there.
The exception should only contain data that explains what was wrong.